highway_env/vehicle/behavior.py

```diff
@@ -377,3 +377,7 @@ class DefensiveVehicle(LinearVehicle):
     ACCELERATION_PARAMETERS = [MERGE_ACC_GAIN / ((1 - MERGE_VEL_RATIO) * MERGE_TARGET_VEL),
                                MERGE_ACC_GAIN / (MERGE_VEL_RATIO * MERGE_TARGET_VEL),
                                2.0]
+
+class ParkingVehicle(DefensiveVehicle):
+    def acceleration(self, ego_vehicle, front_vehicle=None, rear_vehicle=None):
+        return np.random.randint(-2.0,2.0)
```

Comment:
I don't get this part
Aren't these supposed to be motionless (parked) vehicles?
Also, why randint instead of random float?

Reply:
It sure why I did this either. I think I found that the default Defensive Vehicle were accelerating up to highway speeds, so I did this to keep them slow. Probably a stupid way.

Reply:
Some vehicles will be parked, and some will be moving around , so that is why I did not set the acceleration to zero.